src/templates/prefetch.mustache

```diff
@@ -54,12 +52,44 @@
     }
   }
 
+  var language = '{LANG}';
+  var languageKey = 'LANG';
+  if (language !== `{${languageKey}}` && /^([a-z]{2,3})(-([A-Z]{2}|[A-Za-z]{4,5}))?$/.test(language) && hasDynamicContent && dynamicContentFiles) {
+    if (dynamicContentFiles.includes(`{{{localizationPattern}}}`)) {
+      resList.push(`{{{localizationPattern}}}`);
+    } else if (prefetchConfig.fallbackLocalesMap) {
+      var fallback = false;
+      var inputLanguage = language;
+      if (prefetchConfig.fallbackLocalesMap[inputLanguage]) {
+        language = prefetchConfig.fallbackLocalesMap[inputLanguage];
+        fallback = addLocalizationFile(`{{{localizationPattern}}}`);
```

Comment:
unless I am missing something, this condition will always be false
the condition `dynamicContentFiles.includes(`{{{localizationPattern}}}`)` has already been evaluated as false at line 58